.github/workflows/build_julia.yml

```diff
@@ -0,0 +1,127 @@
+name: Julia / OpenCL.jl Tests
+
+permissions:
+  contents: read
+
+on:
+  push:
+    branches:
+      - 'main'
+      - 'release*'
+  pull_request:
+    paths-ignore:
+      - 'doc/**'
+      - 'CHANGES'
+      - 'COPYING'
+      - 'CREDITS'
+      - 'LICENSE'
+      - 'README.*'
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.event.pull_request.number || github.ref || github.run_id }}
+  cancel-in-progress: true
+
+jobs:
+  test_julia:
+    name: Julia ${{ matrix.julia-version }} - LLVM ${{ matrix.llvm }}
+    runs-on: ubuntu-24.04
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+          - julia-version: '1.10'
+            llvm: 15
+          - julia-version: '1.11'
+            llvm: 16
+
+    steps:
+      - name: Checkout PoCL
+        uses: actions/checkout@v4
+        with:
+          path: pocl
+
+      - name: Setup Julia
+        uses: julia-actions/setup-julia@v2
+        with:
+          version: ${{ matrix.julia-version }}
+          arch: x64
+
+      - name: Setup Julia cache
+        uses: julia-actions/cache@v2
+
+      - name: Install system dependencies
+        run: |
+          sudo apt-get update
+          sudo apt-get install -y build-essential cmake ninja-build pkg-config
+
+      - name: Install Julia dependencies
+        run: |
+          touch pocl/Project.toml
+          julia --project=pocl --color=yes -e '
+            using Pkg
+
+            # unversioned
+            Pkg.add([
+              "SPIRV_Tools_jll",
+              "SPIRV_LLVM_Translator_jll",
```

Comment:
does Julia have versioned SPIRV-LLVM-Translator ? if it does, we should probably use it.